### ISSUE TITLE
Event "finish.countdown" dont working

### ISSUE THREAD

**Issue by nonato** [hilios/jQuery.countdown#153]
Hello, I am trying to display an alert on a page that contains 3 countdown.
The alert is to be displayed when any of them is finalized.
But nothing happens when I use "finish.countdown".
It simply resets the counter.
I tested the event "update.countdown" and worked well.
I'm missing something?
Below is my code.
thank you

HTML CODE
```html
<div data-countdown="2015-14-10 09:00:00"></div>
<div data-countdown="2015-14-10 10:00:00"></div>
<div data-countdown="2015-14-10 11:00:00"></div>
```

JAVASCRIPT CODE
```javascript
$('[data-countdown]').each(function() {
  var $this = $(this), finalDate = $(this).data('countdown');

  $this.countdown(finalDate, function(event) {
    $this.html(event.strftime('%H:%M:%S'));
  });

  $(this).on('finish.countdown', function(event){
    alert("teste");
  });
});
```

**Comment by hilios** [comment]
> With the legacy approach one will handle all events in a single callback (update, finish or stop) through the `event.type` property [...]
Source: http://hilios.github.io/jQuery.countdown/documentation.html#introduction

Try not to use the legacy callback with a registered event, the suggested approach is to bind each event one like to handle:

```js
$('[data-countdown]').each(function() {
  var $this = $(this), finalDate = $(this).data('countdown');

  $this.countdown(finalDate)
  $this.on('update.countdown', function(event){
    $this.html(event.strftime('%H:%M:%S'));
  });
  $this.on('finish.countdown', function(event){
    alert("teste");
  });
});
```

**Comment by nonato** [comment]
Hello Hilios, thanks for the reply.

I tried the way you suggested, but does not work.

What I find strange is that if I put the "alert" inside the "update.countdown", it works. 
Every second, appears the "alert: teste update.countdown" on the screen.

Like the code bellow:

```javascript
$('[data-countdown]').each(function() {
  var $this = $(this), finalDate = $(this).data('countdown');

  $this.countdown(finalDate)

  $this.on('update.countdown', function(event){
    $this.html(event.strftime('%H:%M:%S'));
    alert("teste update.countdown");
  });

  $this.on('finish.countdown', function(event){
    alert("teste finish.countdown");
  });
});
```

Every second, appears the "teste update.countdown" on the screen.
But after the counter reaches zero, simply starts again from "23:59:59" and dont shows "teste finish.countdown" alert.

It may be a bug in the latest version?
I downloaded the latest version available.

Thank you very much.

**Comment by hilios** [comment]
Very odd indeed, please try to provide an example with [JSFiddle](https://jsfiddle.net).

It looks like that there is other script modifying the first one, but it's very hard to tell with so little know of your code.

Also, check out the [advanced example](http://hilios.github.io/jQuery.countdown/examples/advanced-coupon-site.html) and change the select to `5 seconds from now (finishin in ...)`, it works like a charm.

**Comment by nonato** [comment]
Thank you for your help Hilios.

I apologize for the delay in responding.

I'm with broken notebook and just not being able to test your suggestion.

I'll get my notebook again this week and I will do the test.

Again, thank you.

**Comment by nonato** [comment]
Hello Hilios, I found out what had happened.
It was a small mistake I made.
The date format was wrong.
When I changed the string that shows the counter to show how many days were lacking for the event, I saw that there were still many days.
For this reason, the counter back to 23:59:00 when reached 00:00:00.
In fact, he was reduced one day to the event.
I was using the format: YYYY-DD-MM.
But I should have used: YYYY-MM-DD.
After correcting this, everything worked properly.
Thank enough for their commitment to help me.
Thank you so much.
Big hug
